Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -27,12 +28,25 @@ public string GetGovernorRoleName(eLookupGovernorRole role, eTextCase textCase =
             return name;
         }
 
-        public string GetEstablishmentsPluralName(GT groupType, eTextCase textCase = eTextCase.TitleCase)
+        public static string GetEstablishmentsPluralName(GT groupType, eTextCase textCase = eTextCase.TitleCase)
         {
-            if (groupType.OneOfThese(GT.ChildrensCentresCollaboration, GT.ChildrensCentresGroup)) return "children's centres".ToTextCase(textCase);
-            else if (groupType.OneOfThese(GT.Federation, GT.Trust)) return "schools".ToTextCase(textCase);
-            else if (groupType.OneOfThese(GT.MultiacademyTrust, GT.SingleacademyTrust, GT.SchoolSponsor)) return "academies".ToTextCase(textCase);
-            else throw new NotImplementedException($"Group type '{groupType}' is not supported for this operation");
+            switch(groupType)
+            {
+                case GT.ChildrensCentresCollaboration:
+                case GT.ChildrensCentresGroup:
+                    return "children's centres".ToTextCase(textCase);
+                case GT.Federation:
+                case GT.Trust:
+                    return "schools".ToTextCase(textCase);
+                case GT.MultiacademyTrust:
+                case GT.SingleacademyTrust:
+                case GT.SchoolSponsor:
+                    return "academies".ToTextCase(textCase);
+                case eLookupGroupType.UmbrellaTrust:
```

Comment:
For consistency, consider: `GT.UmbrellaTrust` ?

Note that further up the file is this, which makes the `GT` / `eLookupGroupType` effectively interchangeable for now:
```csharp
using GT = eLookupGroupType;
```

Reply:
I have changed this - thanks